adaptive_shield/manifest.json

```diff
@@ -0,0 +1,56 @@
+{
+	"manifest_version": "2.0.0",
+	"app_id": "adaptive-shield",
+	"app_uuid":"0c72bf61-1de6-4408-8a24-86f8e3413e07",
```

Comment:
I confirmed that this is the correct `app_uuid` for the published version of the developer platform app that you used to make the oauth client. Thanks for including this!